.env.example

```diff
@@ -1,28 +1,26 @@
 # mandatory environment variables
-INFERENCE_MODEL_ID= ""
+INFERENCE_MODEL_ID= "ibm-granite/granite-3.2-8b-instruct"
 
 # mandatory unless a local deployment is used
-REMOTE_BASE_URL= ""
+REMOTE_BASE_URL= "http://localhost:8321"
 
 # optional environment variables for all demos
-REMOTE=""
-LOCAL_SERVER_PORT=""
-TEMPERATURE=""
-TOP_P=""
-MAX_TOKENS=""
-STREAM=""
+TEMPERATURE="0.0"
+TOP_P="0.95"
+MAX_TOKENS="4096"
+STREAM="True"
```

Comment:
I think for all the notebooks (level 2-6), we have been rendering every step of the agent turn for which `STREAM=False` so can we set the default to `False` instead?